### PR TITLE
Add update and delete actions for MicroPetitions and Comments

### PR DESCRIPTION
**Pull request by igaponov**

### CODE REVIEW

File: backend/src/Civix/CoreBundle/Service/Poll/CommentManager.php [hunk 1]
```diff
@@ -103,15 +103,15 @@ public function addPollRootComment(Question $question, $message = '')
...
     }
 
     public function addComment(BaseComment $comment)
     {
-        return $this->saveNewComment($comment, true);
+        return $this->saveComment($comment, true);
     }
 
-    private function saveNewComment(BaseComment $comment, $notify = false)
+    public function saveComment(BaseComment $comment, $notify = false)
```
Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Service/Poll/CommentManager.php]: @igaponov i just want to make sure this rename will not affect anything else calling `saveComment()` (previously `saveNewComment()` ) ?

**Comment by austinpapp** [comment]
@igaponov looks like we have a conflict. i'll help identify the issues but i'll need you to rebase from develop. thanks

**Comment by austinpapp** [comment]
@igaponov nm. small lib conflict that i resolved.

File: backend/src/Civix/CoreBundle/Service/Poll/CommentManager.php (hunk 1 above)
Comment (igaponov) [inline backend/src/Civix/CoreBundle/Service/Poll/CommentManager.php]: It was private and used only in this class.

Comment (austinpapp) [inline backend/src/Civix/CoreBundle/Service/Poll/CommentManager.php]: you're right. i write too much python! haha. thanks again igor

**Pull request merged**